dspace-api/src/main/java/org/dspace/content/service/EntityService.java

```diff
@@ -0,0 +1,135 @@
+     * @throws SQLException If something goes wrong
+     */
+    EntityType getType(Context context, Entity entity) throws SQLException;
+
+    /**
+     * Returns the list of relations for the given Entity object
+     * @param context   The relevant DSpace context
+     * @param entity    The Entity object for which the list of relationships will be returned
+     * @return The list of relationships for the given Entity object
+     */
+    List<Relationship> getAllRelations(Context context, Entity entity);
+
+    /**
+     * Retrieves the list of relationships, which are attached to the Entity object that is passed along, where the
+     * left item object of each relationship is equal to the Item object of the Entity object that is passed along
+     * @param context   The relevant DSpace context
+     * @param entity    The Entity object to be returned
+     * @return The list of relationships that have the Item in the Entity object as their left item
+     */
+    List<Relationship> getLeftRelations(Context context, Entity entity);
+
+    /**
+     * Retrieves the list of relationships, which are attached to the Entity object that is passed along, where the
+     * right item object of each relationship is equal to the Item object of the Entity object that is passed along
+     * @param context   The relevant DSpace context
+     * @param entity    The Entity object to be returned
+     * @return The list of relationships that have the Item in the Entity object as their right item
+     */
+    List<Relationship> getRightRelations(Context context, Entity entity);
+
+    /**
+     * Retrieves the list of relationships for which their relationshiptype has a left or right label that is
+     * equal to the passed along label String
+     * @param context   The relevant DSpace context
+     * @param label     The label that needs to be in the relationshiptype of the relationship
+     * @return The list of relationships that have a relationshiptype with a left or right label
+     *                  that is equal to the label param
+     * @throws SQLException If something goes wrong
+     */
+    List<Relationship> getRelationsByLabel(Context context, String label) throws SQLException;
+
+    /**
+     * Retrieves the list of relationships that have a relationshiptype that contains the EntityType for the given
+     * Entity
+     * in either the leftEntityType or the rightEntityType variables
+     * @param context   The relevant DSpace context
+     * @param entity    The Entity for which the EntityType should be checked for relationships
+     * @return The list of relationships that each contain a relationshiptype in which there is a right or left
+     * entity type that
+     *                  is equal to the entity type for the given entity
+     * @throws SQLException If something goes wrong
+     */
+    List<RelationshipType> getAllRelationshipTypes(Context context, Entity entity) throws SQLException;
```

Comment:
a paginated method should be provided

Reply:
This has been included in https://jira.duraspace.org/browse/DS-4224